servers/rendering/renderer_canvas_cull.cpp

```diff
@@ -1080,9 +1084,13 @@ void RendererCanvasCull::canvas_item_add_polyline(RID p_item, const Vector<Point
 }
 
 void RendererCanvasCull::canvas_item_add_multiline(RID p_item, const Vector<Point2> &p_points, const Vector<Color> &p_colors, float p_width) {
-	ERR_FAIL_COND(p_points.is_empty() || p_points.size() % 2 != 0);
+	ERR_FAIL_COND(p_points.size() % 2 != 0);
 	ERR_FAIL_COND(p_colors.size() != 1 && p_colors.size() * 2 != p_points.size());
 
+	if (p_points.is_empty()) {
```

Comment:
Same here :)

Reply:
Ooh true.

Reply:
It also prevents errors that shouldn't matter if we're not drawing